app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+
+
+def copy_file(command: str) -> None:
+    names = command.split()
+    if names[1] == names[2] or\
+            len(command.split()) != 3 or\
+            command.split()[0] != "cp":
```

Comment:
Check command length first
Then use unpacking like `command, source_file, target_file = names`
Then check command name and files

Reply:
Done